benchmark/package.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
strange to have another package.json for the benchmark without a proper mono-repo structure